package/MDAnalysis/analysis/pca.py

```diff
@@ -398,7 +400,7 @@ def transform(self, atomgroup, n_components=None, start=None, stop=None,
         if isinstance(atomgroup, Universe):
             atomgroup = atomgroup.atoms
 
-        if(self._n_atoms != atomgroup.n_atoms):
+        if (self._n_atoms != atomgroup.n_atoms):
```

Comment:
This formatting change is ok as it was really, really ugly ;-).

Also remove the parentheses.
```suggestion
        if self._n_atoms != atomgroup.n_atoms:
```

Reply:
Done